src/main/java/io/fabric8/elasticsearch/plugin/acl/BaseRolesSyncStrategy.java

```diff
@@ -19,14 +19,15 @@
 import static io.fabric8.elasticsearch.plugin.acl.SearchGuardRoles.USER_KIBANA_PREFIX;
 import static io.fabric8.elasticsearch.plugin.acl.SearchGuardRoles.USER_PREFIX;
 
+import io.fabric8.elasticsearch.plugin.KibanaIndexMode;
 import io.fabric8.elasticsearch.plugin.OpenshiftRequestContextFactory;
 import io.fabric8.elasticsearch.plugin.OpenshiftRequestContextFactory.OpenshiftRequestContext;
 
 public abstract class BaseRolesSyncStrategy implements RolesSyncStrategy {
 
     protected SearchGuardRoles roles;
     private final String userProfilePrefix;
-   
+    
```

Comment:
extraneous space

Reply:
I believe this is simply a space between the member  variables and the constructor